include/prereq-build.mk

```diff
@@ -175,23 +175,21 @@ $(eval $(call SetupHostCommand,install,Please install GNU 'install', \
 $(eval $(call SetupHostCommand,perl,Please install Perl 5.x, \
 	perl --version | grep "perl.*v5"))
 
-$(eval $(call SetupHostCommand,python,Please install Python >= 3.6, \
+$(eval $(call SetupHostCommand,python,Please install Python >= 3.7, \
```

Comment:
I would prefer if we keep python 3.6 support in 23.05.

Reply:
Ack. Removed the commit dropping support for Python 3.6.